drvpptx.cpp: In member function 'Point drvPPTX::pathCentroid()':
drvpptx.cpp:1335:8: error: 'unique_ptr' is not a member of 'std'
 1335 |   std::unique_ptr<Point[]>  allKnots ( new Point[numElts + 1]);
      |        ^~~~~~~~~~
drvpptx.cpp:68:1: note: 'std::unique_ptr' is defined in header '<memory>'; did you forget to '#include <memory>'?
   67 | #include <zip.h>
  +++ |+#include <memory>
   68 | 
drvpptx.cpp:1335:24: error: expected primary-expression before '[' token
 1335 |   std::unique_ptr<Point[]>  allKnots ( new Point[numElts + 1]);
      |                        ^
drvpptx.cpp:1335:25: error: expected primary-expression before ']' token
 1335 |   std::unique_ptr<Point[]>  allKnots ( new Point[numElts + 1]);
      |                         ^
drvpptx.cpp:1335:29: error: 'allKnots' was not declared in this scope
 1335 |   std::unique_ptr<Point[]>  allKnots ( new Point[numElts + 1]);
      |                             ^~~~~~~~
Index: pstoedit-3.78/src/drvpptx.cpp
===================================================================
--- pstoedit-3.78.orig/src/drvpptx.cpp
+++ pstoedit-3.78/src/drvpptx.cpp
@@ -39,6 +39,7 @@
 
 #include <errno.h>
 #include <algorithm>
+#include <memory>
 
 #ifdef _MSC_VER
 // MS VC++ Windows
